api/identityfile/identityfile.go

```diff
@@ -301,3 +302,25 @@ func decodeIdentityFile(idFile io.Reader) (*IdentityFile, error) {
 	}
 	return &ident, nil
 }
+
+// currently we only use "CertAlgoRSAv01" and "CertAlgoECDSA256v01",
+// but the full list has been created for completeness.
+var sshCertTypes = []string{
+	ssh.CertAlgoRSAv01,
+	ssh.CertAlgoDSAv01,
+	ssh.CertAlgoECDSA256v01,
+	ssh.CertAlgoECDSA384v01,
+	ssh.CertAlgoECDSA521v01,
+	ssh.CertAlgoSKECDSA256v01,
+	ssh.CertAlgoED25519v01,
+	ssh.CertAlgoSKED25519v01,
+	ssh.CertAlgoRSASHA256v01,
+	ssh.CertAlgoRSASHA512v01,
```

Comment:
Looks like these two should never actually appear as on-disk certs, FYI.

Should we check for `-cert-v.*@openssh.com` instead?

Reply:
Sure, I've updated it to use a regex